js/scripts.js

```diff
@@ -163,12 +163,32 @@ for (let i = 0; i < prjData.length; i += 1) { /* Creating project cards using th
 }
 
 const formData = {
+  fullname: '',
   email: '',
+  textarea: '',
 };
 
+function getText(val) {
+  formData[val.target.id] = val.target.value;
+}
+
+const fullname = document.querySelector('#fullname');
 const email = document.querySelector('#email');
-email.addEventListener('change', (e) => {
-  formData[e.target.id] = e.target.value;
+const textarea = document.querySelector('#textarea');
+
+fullname.addEventListener('input', getText);
+email.addEventListener('input', getText);
+textarea.addEventListener('input', getText);
+
+fullname.value = localStorage.getItem('fullname');
+email.value = localStorage.getItem('email');
+textarea.value = localStorage.getItem('textarea');
+
+contact.addEventListener('input', () => {
+  const { fullname, email, textarea } = formData;
+  localStorage.setItem('fullname', fullname);
+  localStorage.setItem('email', email);
+  localStorage.setItem('textarea', textarea);
 });
```

Comment:
- [ ] Hello there, team. So far, you've done well. However, given the requirement, I would recommend that the data be saved in an object format in local storage rather than separately.

#### Requirement :point_down: 
![Screenshot from 2023-05-18 22-53-14](https://github.com/lRebornsl/portfolio/assets/89228743/dcd295b7-1284-458d-9313-f44e19d364f9)

#### The result of your work so far :point_down: 
![Screenshot from 2023-05-18 22-52-13](https://github.com/lRebornsl/portfolio/assets/89228743/84483a90-2eac-4607-9c57-5baeeb84d1bf)

You can update your code like this :point_down: 
```
// Retrieve data from local storage or initialize with empty values
const storedData = JSON.parse(localStorage.getItem('formData')) || {
  fullname: '',
  email: '',
  textarea: '',
};

// Assign the stored data to the formData object
const formData = {
  ...storedData
};

// Update the formData object and store it in local storage
function getText(val) {
  formData[val.target.id] = val.target.value;
  localStorage.setItem('formData', JSON.stringify(formData));
}

const fullname = document.querySelector('#fullname');
const email = document.querySelector('#email');
const textarea = document.querySelector('#textarea');

// Assign the stored values to the input fields
fullname.value = formData.fullname;
email.value = formData.email;
textarea.value = formData.textarea;

// Add event listeners to update formData and store it in local storage on input change
fullname.addEventListener('input', getText);
email.addEventListener('input', getText);
textarea.addEventListener('input', getText);

```

Reply:
Thanks a lot for your feedback!
My bad for not having seen that part of the task 😞 
I didn't know about JSON, so thanks for the data.